src/components/ButtonPanel.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Button from './Button';
+
+function ButtonPanel() {
+  return (
+    <div>
+      <div>
+        <Button buttonName="AC" />
+        <Button buttonName="+/-" />
+        <Button buttonName="%" />
+        <Button buttonName="÷" />
+      </div>
+      <div>
+        <Button buttonName="7" />
+        <Button buttonName="8" />
+        <Button buttonName="9" />
+        <Button buttonName="X" />
+      </div>
+      <div>
+        <Button buttonName="4" />
+        <Button buttonName="5" />
+        <Button buttonName="6" />
+        <Button buttonName="-" />
+      </div>
+      <div>
+        <Button buttonName="1" />
+        <Button buttonName="2" />
+        <Button buttonName="3" />
+        <Button buttonName="+" />
+      </div>
+      <div>
+        <Button buttonName="0" />
+        <Button buttonName="." />
+        <Button buttonName="=" />
+      </div>
```

Comment:
### [Optional]
I think we could make use of two loops to make the Button instead of repeating ourselves. let's practice [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) code.

To do this; create an array of symbols and loop through the array using [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.